tests/strategies/order_strategy/test_order_cluster.py

```diff
@@ -8,8 +10,8 @@
 
 
 class TestOrderCluster:
-    @pytest.fixture
-    def basic_order_cluster(self) -> OrderCluster:
+    @pytest.fixture(name="basic_order_cluster")
+    def _get_basic_order_cluster(self) -> Iterator[OrderCluster]:
```

Comment:
I'm not convinced adding attribute `name` to the fixture instead of referring to the name of the fixture itself adds value. On the contrary, it migth obfuscate references.

Reply:
We have been already applying this style in the `vrtool` project and it is preferred so that the methods are more explicit ( `get` / `factory` / ...) about what they do and the name easier to reference as the arguments become shorter.